cypress/integration/maintenance/index.js

```diff
@@ -2,17 +2,17 @@ import { Before, Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
 
 Before(() => {
     cy.intercept({ pathname: /maintenance$/, method: 'POST' }, req => {
-        const formData = new TextDecoder().decode(req.body)
+        const url = new URL(req.url)
```

Comment:
The `POST /maintenance` API supports both URL query params and encoded form params. Switching to URL query params by using `useDataQuery` allows us to simplify this test.